src/common/constants.ts

```diff
@@ -318,13 +319,13 @@ export const monthAndDayFormat = 'MMMM d';
 export const yearMonthAndDayFormat = 'yyyy MM d';
 
 export const orderStatus = {
-  NEW: 'New order',
-  REJECTED: 'Rejected',
-  SENT: 'Sent',
-  RECEIVED: 'Received',
-  SENT_BACK: 'Sent back',
-  OVERDUE: 'Overdue',
-  RETURNED: 'Returned',
+  NEW: OrderStatus.NEW,
+  REJECTED: OrderStatus.REJECTED,
```

Comment:
why you redefine the same constants? the only difference is the name of this constants OrderStatus and orderStatus and one of them is enum. may be we can delete one of them?

Reply:
deleted from constants, but it was used all over the project, had to change it to enum